base/doc/ltnews33.tex

```diff
@@ -490,6 +490,19 @@ \subsection{Extending \pkg{latexrelease} to declare an entire module}
 
 
 
+\subsection{Small fix for rolling back prior to 2020-02-02}
+
+Whereas the \pkg{latexrelease} package can usually emulate an older \LaTeX{}
+kernel without much problem, rolling back to before the 2020-02-02 release
+didn't work properly because the management of the \cs{ExplSyntaxOn/Off} status
+for packages cannot be removed by the rollback without messing up catcodes after
+an \pkg{expl3}-based package is loaded.  This has been fixed and now rollback is
+more careful not to leave \pkg{ExplSyntaxOn}afterapackageends.
```

Comment:
as already mentioned

Reply:
That was supposed to be a small joke, but it seems it didn't catch on :)

Fixed this and the version number.

Reply:
I was wondering about that briefly but then I know we will get it corrected by Barbara for the TUB version anyway :-)

Reply:
> That was supposed to be a small joke, but it seems it didn't catch on :)

I couldn't imagine a joke here :wink: But it is indeed, and a funny one! :smile:

Reply:
well mabye you leave it :-)

Reply:
Without a smiley (or any other bell), high is the probability that people see this as a typo :smile:

Reply:
\footnote{\ldots with this result.}